sass/DynamicHierarchicalFacetSearch/_DynamicHierarchicalFacetSearch.scss

```diff
@@ -0,0 +1,43 @@
+@import '../Variables';
+
+.coveo-dynamic-hierarchical-facet-search-value {
+  display: flex;
+  flex-direction: column;
+  padding: 5px;
+  cursor: pointer;
+
+  .coveo-dynamic-hierarchical-facet-search-value-results-count {
+    margin-left: 5px;
+    color: $color-light-contrast-grey;
+    font-weight: lighter;
+  }
+
+  .coveo-dynamic-hierarchical-facet-search-value-path {
+    padding: 0;
+    color: $color-light-contrast-grey;
+    font-weight: lighter;
+
+    display: flex;
+
+    .coveo-dynamic-hierarchical-facet-search-value-path-ellipsis,
+    .coveo-dynamic-hierarchical-facet-search-value-path-part {
+      display: inline;
+    }
+
+    .coveo-dynamic-hierarchical-facet-search-value-path-part {
+      flex: 0 15 auto;
+      min-width: 10px;
+      @supports (max-width: max-content) {
+        flex: 1 0 0;
+        max-width: max-content;
+      }
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
Although only the first and last two path parts are displayed, I added this CSS in case they would be too long. This weird bit of CSS adds an ellipsis to longer values first.

For instance:
`I believe this is a longer phrase/hello` could be shortened to `I believe thi.../hello` or `I be.../hell...`.